projects/packages/forms/src/blocks/field-file/edit.js

```diff
@@ -0,0 +1,64 @@
+import { useBlockProps, useInnerBlocksProps } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+import clsx from 'clsx';
+import JetpackFieldControls from '../shared/components/jetpack-field-controls';
+import useFormWrapper from '../shared/hooks/use-form-wrapper';
+import useJetpackFieldStyles from '../shared/hooks/use-jetpack-field-styles';
+import './editor.scss';
+
+export default function FileFieldEdit( props ) {
+	const { attributes, clientId, isSelected, setAttributes, name, className } = props;
+	const { id, required, width } = attributes;
+
+	useFormWrapper( { attributes, clientId, name } );
+	const { blockStyle } = useJetpackFieldStyles( attributes );
+
+	let classes = clsx( className, 'jetpack-field is-non-animated-label', {
+		'is-selected': isSelected,
+	} );
+	if ( width ) {
+		classes += ` jetpack-field__width-${ width }`;
+	}
```

Comment:
Untested but suppose this should work?

```suggestion
	let classes = clsx( className, 'jetpack-field is-non-animated-label', {
		'is-selected': isSelected,
		`jetpack-field__width-${ width }`: width,
	} );
```

Reply:
I tried this but it didn't work.

Reply:
Then no biggie, current way is fine.

Reply:
Or it might need brackets to work 🤔  
```js
[ `jetpack-field__width-${ width }` ]: width,
```